s3-connector/.gitattributes

```diff
@@ -3,4 +3,7 @@
 #
 # These are explicitly windows files and should use crlf
 *.bat           text eol=crlf
-
+*.txt           text eol=lf
+*.md            text eol=lf
+*.java          text eol=lf
+*.kts           text eol=lf
```

Comment:
@jjaakola-aiven added to enforce the line feed at the end before pushing to git

Reply:
Unfortunately seems we cannot do that: https://github.com/Aiven-Open/commons-for-apache-kafka-connect/actions/runs/10793874906/job/29936840957?pr=285